tools/install_demo_configuration.sh

```diff
@@ -372,6 +372,8 @@ echo "plugins.security.audit.type: internal_opensearch" | $SUDO_CMD tee -a "$OPE
 echo "plugins.security.enable_snapshot_restore_privilege: true" | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
 echo "plugins.security.check_snapshot_restore_write_privileges: true" | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
 echo 'plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
+echo "plugins.security.restapi.password_validation_regex: '(?=.*[A-Z])(?=.*[^a-zA-Z\d])(?=.*[0-9])(?=.*[a-z]).{8,}'" | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
+echo 'plugins.security.restapi.password_validation_error_message: "A password must be at least 8 characters long and contain at least one uppercase letter, one lowercase letter, one digit, and one special character."' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
```

Comment:
seems wrong. Error message should not be added to opensearch.yml file...

Reply:
I copied the format in the example file here: [](https://github.com/opensearch-project/security/blob/main/securityconfig/opensearch.yml.example#L180)

Reply:
I see, it's already this way in the code. thx